ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.18__amd64-nightly-4.18-cpou-upgrade-from-4.16.yaml

```diff
@@ -59,6 +63,54 @@ resources:
       cpu: 100m
       memory: 200Mi
 tests:
+- as: aws-ipi-ovn-hypershift-mce-inplace-f28
+  cron: 2 9 2 * *
+  steps:
+    cluster_profile: aws-qe
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      EXTRA_ARGS: --node-upgrade-type=InPlace
+      MCE_VERSION: "2.6"
+      TEST_FILTERS: ~DisconnectedOnly&;~HyperShiftMGMT&;~MicroShiftOnly&;~NonHyperShiftHOST&
+    test:
+    - chain: openshift-upgrade-qe-test-hypershift
+    workflow: cucushift-installer-rehearse-aws-ipi-ovn-hypershift-mce-guest
+- as: aws-ipi-ovn-hypershift-mce-replace-f28
+  cron: 44 15 4 * *
+  steps:
+    cluster_profile: aws-qe
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      MCE_VERSION: "2.8"
+      TEST_FILTERS: ~DisconnectedOnly&;~HyperShiftMGMT&;~MicroShiftOnly&;~NonHyperShiftHOST&
+    test:
+    - chain: openshift-upgrade-qe-test-hypershift
```

Comment:
The job file `-cpou-` is for [control-plane-only update](https://docs.openshift.com/container-platform/4.18/updating/updating_a_cluster/control-plane-only-update.html). But this chain seems not a cpou test. I guess you could host the profile in a 2-hops chain upgrade job config file, such as `openshift-openshift-tests-private-release-4.18__amd64-nightly-4.18-upgrade-from-stable-4.16.yaml`.